pages/invitation/InvitationHandler.php

```diff
@@ -117,256 +110,29 @@ public static function getActionUrl(InvitationAction $action, Invitation $invita
-    }
-
-    /**
-     * Edit user using user access table action
-     * @param $args
-     * @param $request
-     * @return void
-     * @throws \Exception
-     */
-    public function editUser($args, $request): void
-    {
-        $invitation = null;
-        if(!empty($args)) {
-            $invitationMode = 'editUser';
-            $invitationData = $this->generateInvitationPayload($args[0],[],$request->getContext());
-            $user = $invitationData['user'];
-            $invitationPayload = $invitationData['invitationPayload'];
-            $templateMgr = TemplateManager::getManager($request);
-            $breadcrumbs = $templateMgr->getTemplateVars('breadcrumbs');
-            $this->setupTemplate($request);
-            $context = $request->getContext();
-            $breadcrumbs[] = [
-                'id' => 'contexts',
-                'name' => __('navigation.access'),
-                'url' => $request
-                    ->getDispatcher()
-                    ->url(
-                        $request,
-                        Application::ROUTE_PAGE,
-                        null,
-                        'management',
-                        'settings',
-                        ['access']
-                    )
-            ];
-            $breadcrumbs[] = [
-                'id' => 'invitationWizard',
-                'name' => __('invitation.wizard.pageTitle'),
-            ];
-            $steps = new SendInvitationStep();
-            $templateMgr->setState([
-                'steps' => $steps->getSteps($invitation, $context,$user),
-                'emailTemplatesApiUrl' => $request
-                    ->getDispatcher()
-                    ->url(
-                        $request,
-                        Application::ROUTE_API,
-                        $context->getData('urlPath'),
-                        'emailTemplates'
-                    ),
-                'primaryLocale' => $context->getData('primaryLocale'),
-                'invitationType' => 'userRoleAssignment',
-                'invitationPayload' => $invitationPayload,
-                'invitationMode' => $invitationMode,
-                'pageTitle' =>
-                    $invitationPayload['givenName'][Locale::getLocale()] . ' '
-                    . $invitationPayload['familyName'][Locale::getLocale()],
-                'pageTitleDescription' =>
-                    __(
-                        'invitation.wizard.viewPageTitleDescription',
-                        ['name' => $invitationPayload['givenName'][Locale::getLocale()]]
-                    ),
-            ]);
-            $templateMgr->assign([
-                'pageComponent' => 'Page',
-                'breadcrumbs' => $breadcrumbs,
-                'pageWidth' => TemplateManager::PAGE_WIDTH_FULL,
-            ]);
-            $templateMgr->display('/invitation/userInvitation.tpl');
+        $invitationType = $args[0]; // invitation type
```

Comment:
Here, the way it is developed, I think there is no need for both args, right? either the type or the id is needed.